olp-cpp-sdk-authentication/include/olp/authentication/Settings.h

```diff
@@ -23,6 +23,7 @@
 #include <memory>
 #include <string>
 
+#include <olp/authentication/AuthenticationCredentials.h>
```

Comment:
should be separate commit for OLPEDGE-839 ticket.
should be before #include "AuthenticationApi.h", just #include "AuthenticationCredentials.h"

Reply:
aligned with the ticket you mentioned for rebase ease.